packages/local_auth/local_auth_android/android/build.gradle

```diff
@@ -31,13 +31,13 @@ android {
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
```

Comment:
I changed it because it was set in the build.gradle of other pacakges. Now that I think about it, it's an unrelated change, so I think I'll change it back.

Reply:
Adding it is also a no-op (which is why I didn't add it when I added `sourceCompatibility`); the default value of `targetCompatibility` is `sourceCompatibility`'s value.